ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.4-presubmits.yaml

```diff
@@ -44,3 +44,47 @@ presubmits:
         secret:
           secretName: sentry-dsn
     trigger: (?m)^/test( | .* )build,?($|\s.*)
+  - agent: kubernetes
```

Comment:
@petr-muller 
How come we got this one?
Is it because we have config for ci-operator but not the job def. for prow?

Reply:
1. First, https://github.com/openshift/release/pull/5156 merged - it added the new test into configs for `master`, `release-4.2` and `release-4.3` and generated jobs for this new test for all three branches.
2. Second, `config-brancher` noticed this new test is in `master` config but not in `release-4.4` config, and added it to the 4.4 one. https://github.com/openshift/release/pull/5113/files#diff-90e55e1affd1936d98df89685c4d98d5R30-R33
3. Third, prowgen noticed there is a test in release-4.4 config without a corresponding job, so it generated it.

Reply:
Clear. Thanks.
My missing part is "Second": master to the future release.